openwisp_users/migrations/0003_default_organization.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.1.1 on 2018-09-16 12:20
+
+from django.db import migrations
+from django.conf import settings
+
+
+def set_default_organization_uuid(apps, schema_editor):
+    """
+    Get default organzation or create a default
+    organization and set settings.DEFAULT_ORG_UUID
+    """
+    organization = apps.get_model('openwisp_users', 'organization')
+    try:
+        default_organization = organization.objects.get(
+            slug=settings.DEFAULT_ORG_SLUG)
+
+    except (AttributeError, organization.DoesNotExist):
+        default_organization = organization.objects.create(
+            name='default',
+            slug='default',
+            is_active=False,
+            description='This is the default organization. '
+            'It created because no organization was set as default '
+            'during setup. If you want to delete it, please make sure '
+            'all the users are moved to appropriate organization '
+            'because when an organization is delete all the '
+            'organization users are deleted as well!',
+        )
+
+    settings.DEFAULT_ORG_SLUG = default_organization.slug
+    settings.DEFAULT_ORG_UUID = default_organization.pk
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('openwisp_users', '0002_auto_20180508_2017'),
```

Comment:
The migration `('openwisp_users', '0002_auto_20180508_2017')` is a dependency in openwisp-ipam only as of now, since ipam has not been released is it possible to rename this dependency's name to something more descriptive?

Reply:
unfortunately not, that was already released :(

Reply:
unfortunately this was already released and I only noticed afterwards.
We could add a GCI task to write a script which checks that the last migration does not create `_auto_` and once we get it working in one of the django modules we can add it to all of them, what do you think?